### PR TITLE
all: Correct use of time

### PR DESCRIPTION
**Pull request by aDotInTheVoid**
Means the config is readable, and should fix dead-machine-monitor.

r? @danielg0 

cc @Euphoride

### CODE REVIEW

File: internal/config/duration.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package config
+
+import "time"
+
+type Duration time.Duration
+
+const (
```
Comment (Euphoride) [inline internal/config/duration.go:7]: NIT/Curiosity: Why redefine?

**Review by Euphoride — COMMENTED** [review]
LGTM dependent on CI passing - some churn on variable naming but readable PR anyhow

File: internal/config/duration.go (hunk 1 above)
Comment (aDotInTheVoid) [reply internal/config/duration.go:7]: So that their of the right type.

**Review by danielg0 — APPROVED** [review]
LGTM, @aDotInTheVoid, tests pass now. Do you mind if I make the couple of other changes needed to remove the other references to Unixtimes in postgres.go, or do you want to leave this to another pr?

**Comment by aDotInTheVoid** [comment]
Let’s do that in a follow up, this is already kinda big

**Pull request merged**